package.json

```diff
@@ -1,34 +1,63 @@
 {
   "name": "2020q3",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+  "description": "ReactJS Global Mentoring Program 2020Q3",
+  "version": "1.0.0",
+  "author": "Hvozdzeu Aliaksandr <Aliaksandr_Hvozdzeu@epam.com>",
+  "license": "ISC",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gvozdev1986/ReactJS-Global-Mentoring-Program-2020Q3.git"
   },
+  "keywords": [
+    "react-js"
+  ],
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "webpack-dev-server --open --hot",
+    "build:prod": "set env ENV=prod && webpack",
+    "build:dev": "set env ENV=dev && webpack",
+    "lint": "eslint ./src --ext .js,.jsx",
+    "test": "jest --no-cache --coverage --config jest.config.json"
   },
-  "eslintConfig": {
-    "extends": "react-app"
+  "dependencies": {
+    "@babel/runtime": "^7.10.5",
+    "@jest/globals": "^26.1.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
```

Comment:
please read about deps and devDeps

Reply:
and meaning of symbols before versions

Reply:
> please read about deps and devDeps

Thanks for your note. I have read and understand what the difference between them.

Reply:
> and meaning of symbols before versions

The same. Thanks for your note.